config/community_product_definitions.json

```diff
@@ -0,0 +1,499 @@
+{
+  "ps_products": {
+    "community": {
+      "name": "Community Projects",
+      "team": "",
+      "ps_modules": [
+        "fedora-34",
+        "fedora-35",
+        "fedora-36",
+        "fedora-37",
+        "fedora-all",
+        "fedora-rawhide",
+        "epel-7",
+        "epel-8",
+        "epel-all",
+        "openstack-rdo"
+      ],
+      "business_unit": "Community",
+      "errata_product_tags": []
```

Comment:
Are we trying to keep parity of this data to our internal version? I guess we could strip all these attributes that don't really matter for the community version.

Reply:
I really just wanted a copy of the data without having to rely on the dashboard service, so that we didn't have any reliance on internal services. 

I did remove all the unused attributes now.